draft-ietf-rats-corim.md

```diff
@@ -1254,6 +1254,17 @@ are acceptable states.
 Integrity Registers can be used to model the PCRs in a TPM or vTPM, in which case the identifier is the register index, or other kinds of vendor-specific measured objects.
 
 
+##### Raw Integral {#sec-comid-raw-integral}
+
+A raw integral describes an integral value that can be compared with linear order in the target environment.
+A raw integral is represented with either major type 0 or major type 1 ints, or even tagged bignums.
```

Comment:
What is the use case for bignums?

Reply:
I have this question also. For example, when would someone use bigint vs. raw-value (which is bytes also) assuming a future version of raw-value could introduce a range of raw-value?